extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -376,7 +378,16 @@ export class SchemaCompareDialog {
 			return undefined;
 		}
 
+		let count = -1;
+		let idx = -1;
 		let values = cons.map(c => {
+			count++;
+			if (idx === -1) {
```

Comment:
>if (idx === -1) { [](start = 3, length = 17)

I don't think this check is necessary - IDs should be unique and even if they aren't arbitrarily picking the first one isn't really any different than picking the last.